src/test/java/Homework18.java

```diff
@@ -6,18 +6,22 @@
 public class Homework18 extends BaseTest {
 
     @Test
-    public void playASongTest() {
-        login("demo@class.com", "te$t$tudent");
+    public void playASongTest() throws InterruptedException{
+        login("simplyd2d@gmail.com", "te$t$tudent");
+        Thread.sleep(3000);
```

Comment:
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.